### PR TITLE
Add operationId to be shown for each operation

### PR DESCRIPTION
**Pull request by alexkravets**
Puts operationId to the right in every operation header.

### CODE REVIEW

**Comment by shockey** [comment]
@Minasokoni - please review the design 😄

File: src/core/components/operation.jsx [hunk 1]
```diff
@@ -158,7 +160,7 @@ export default class Operation extends React.Component {
                   { summary }
                 </div>
             }
-
+            <span className="opblock-summary-operation-id">{operationId}</span>
```
Comment (shockey) [inline src/core/components/operation.jsx]: Can we put this in a ternary operator? Something like `{ operationId ? <span className="opblock-summary-operation-id">{operationId}</span> : null }`. Will keep the DOM cleaner when an operationId isn't present.

Comment (alexkravets) [inline src/core/components/operation.jsx]: @symonenkov please address this.

**Comment by alexkravets** [comment]
Updated.

**Comment by webron** [comment]
@alexkravets Thanks for the PR, much appreciated.

A couple of points:
- I feel the reduced font size for the path and the summary is too much. Would appreciate if that change was reverted.
- As in #2568, the display of the `operationId` should be _optional_, and should default to `false`. This means there should be a new configuration option, and the README also needs to be update to reflect that.

After those, once @shockey approves the code changes, we'll merge.

@shockey - one question - will https://github.com/swagger-api/swagger-js/pull/1048 affect this? Ideally, if there's no `operationId`, it should not be displayed (rather than display the one we generate for it).

**Comment by shockey** [comment]
@webron operations without explicit IDs will not have their generated ones displayed - we don't hand those back to UI as of today.

There's another edge case here: operation IDs with special characters will be displayed as-is (i.e. we won't show the escaped version that Swagger-Client eventually ends up with). See below:

image

Is this what we want?

**Comment by shockey** [comment]
Looks like we're going to merge https://github.com/swagger-api/swagger-js/pull/1068, so if we want to display unaltered operationIds, the display needs to target `__originalOperationId` from the resolved spec.

**Comment by alexkravets** [comment]
@webron 

1. We've not changed size of the summary, path font size has been changed cause `operationId` is using bigger font size. We've made this change as `operationId` is a higher abstraction layer while operation path is implementation detail (we use spec to execute operations and operations path/method are not used at all).

2. Why `operationId` should default to `false`?

**Comment by webron** [comment]
1. This is a documentation tool, and the path is a critical part for the user, more so than the operationId. I've checked out the result, and it's just too small. Please revert it.

2. Because it was `false` in the previous version, and most users will not want to display it. I'm happy to re-add the option to show it, and anyone who wants to use it will just enable it.

**Comment by shockey** [comment]
Merged! The feature will go live on Friday when we do our weekly release.

Thanks for the contribution, @alexkravets. We appreciate it, and look forward to seeing you in our Pull Requests again, time permitting 😄

**Comment by alexkravets** [comment]
Great! Thank you.

**Pull request merged**